components/list/list-item.js

```diff
@@ -125,200 +40,52 @@ class ListItem extends RtlMixin(LitElement) {
 				margin-top: 1.15rem;
 			}
 
-			.d2l-list-item-main {
-				flex-grow: 1;
-				margin-top: 0.05rem;
-			}
-
-			.d2l-list-item-content-flex ::slotted([slot="actions"]) {
-				align-self: flex-start;
-				display: grid;
-				flex-grow: 0;
-				grid-auto-columns: 1fr;
-				grid-auto-flow: column;
-				grid-gap: 0.3rem;
-				margin: 0.15rem 0;
-				z-index: 4;
-			}
-
-			a, label {
-				height: 100%;
-				position: absolute;
-				width: 100%;
-				z-index: 2;
-			}
-
-			:host([href]) label {
-				width: 2.1rem;
-				z-index: 3;
-			}
-
-			:host([href]) {
-				--d2l-list-item-content-text-color: var(--d2l-color-celestine);
-			}
-
-			a[href]:focus + .d2l-list-item-content,
-			a[href]:hover + .d2l-list-item-content {
-				--d2l-list-item-content-text-decoration: underline;
-			}
-
-			:host([href]) .d2l-list-item-link:focus {
-				outline: none;
-			}
-
-			.d2l-list-item-container[breakpoint="1"] ::slotted([slot="illustration"]),
-			.d2l-list-item-container[breakpoint="1"] .d2l-list-item-content-flex ::slotted([slot="illustration"]) {
+			.d2l-list-item-container[breakpoint="1"] ::slotted([slot="illustration"]) {
 				margin-right: 1rem;
 				max-height: 3.55rem;
 				max-width: 6rem;
 			}
 
-			:host([dir="rtl"]) .d2l-list-item-container[breakpoint="1"] ::slotted([slot="illustration"]),
-			:host([dir="rtl"]) .d2l-list-item-container[breakpoint="1"] .d2l-list-item-content-flex ::slotted([slot="illustration"]) {
+			:host([dir="rtl"]) .d2l-list-item-container[breakpoint="1"] ::slotted([slot="illustration"]) {
```

Comment:
`:host([dir="rtl"]) .d2l-list-item-container[breakpoint="1"] ::slotted([slot="illustration"])`
Giving access to the breakpoints is awesome. I'm not too happy with how this works. `.d2l-list-item-container[breakpoint="1"]` requires too much information from the mixin. Maybe this isn't such a big problem.

I could also rename `.d2l-list-item-container[breakpoint="1"]` to `.d2l-list-item-breakpoint[breakpoint="1"]` which is a little more clear. I would just add the  `.d2l-list-item-breakpoint` class to `.d2l-list-item-container` and it would have both which gets messy on the mixin side.